src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,23 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
+            limit: Joi.string(),
```

Comment:
O campo ``limit deveria ser um número inteiro, aqui no Joi seria o lugar certo para validar isso e já definir o valor padrão caso o usuário não informasse nada.
```suggestion
            limit: Joi
                .number()
                .integer()
                .positive()
                .max(50)
                .default(50),
```